### ISSUE TITLE
Home Assistant Config Check Error

### ISSUE THREAD

**Issue by Resinchem** [custom-components/youtube#46]
**Version of the custom_component**
0.8.0

**Describe the bug**
For at least the past couple Home Assistant versions, the YouTube component fails the configuration check.  This probably started with HA 2021.10 (but might have been earlier), but is still present in the current check against HA 2021.11.2.  It doesn't appear to impact the actual upgrade, and as far as I can tell the YouTube add-on still functions after the upgrade (at least to 2021.10.7)

**log**
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[07:27:10] INFO: Don't worry, this temporary installation is not overwriting your current one.
[07:27:10] INFO: Installing Home Assistant: latest...
[07:27:10] INFO: Please be patient, this might take a few minutes...
[07:28:02] INFO: Installed Home Assistant 2021.11.2
[07:28:02] INFO: Making a copy of your configuration for checking...
[07:28:02] INFO: Checking your configuration against this version...
[07:30:00] ERROR: The configuration check did not pass!
[07:30:00] ERROR: See the output below for more details.
Testing configuration at /tmp/config
Failed config
  General Errors: 
    - Platform error sensor.youtube - No module named 'dateutil'

Successful config (partial)
[07:30:00] INFO: The full output has been written to /share/check_config.txt
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

**Comment by pinkywafer** [comment]
unable to reproduce. More info required on install method etc?  I assume this is from the Config check addon? if so, are you running the latest version of the addon?

**Comment by Resinchem** [comment]
Yes, this is shown in the Check Configuration tool and I can confirm it is still present.  I just ran the Check Config on the upgrade from HA Core 2021.11.4 to 2021.12.3 and the logs still show this error:

Failed config
  General Errors: 
    - Platform error sensor.youtube - No module named 'dateutil'

I am running the 0.8.0 version of the YouTube add-on and it was installed via HACS.  I am also running the latest version of the Home Assistant Check Configuration tool - 3.9.0. Everything is currently up to date (except for the most recent Core upgrade).

Note that this does not appear to impact the actual upgrade, nor does it appear to hinder the operation of the YouTube add-on.... at least not in the prior upgrade from 2021.10 to 2021.11 (when the error first appeared in the upgrade check).  I only created the issue because it does seem to prevent a "clean" configuration check prior to upgrade.  But the add-on/sensors still appear to work normally.
